frontend/src/views/web/our-team.html

```diff
@@ -1,124 +1,69 @@
 <div class="white-bg view-container">
     <main-header></main-header>
     <div class="grad-container">
-        <h4 class="team-heading">Team Awesome!</h4>
+        <h4 class="team-heading">{{ourTeam.coreTeamType}}</h4>
         <div class="row">
-            <!-- integrate this card with API -->
-            <div class="col s12 l3 ">
+            <!-- This card is for Core Team -->
+            <div class="col s12 l3" ng-repeat="data in ourTeam.coreTeamDetails">
                 <div class="team-card">
                     <div class="team-link-head">
                         <span>
-    				<a href="http://deshraj.github.io/"><i class="fa fa-linkedin"></i></a>
-    			</span>
+                    <a href="{{data.linkedin_url}}"><i class="fa fa-linkedin"></i></a>
```

Comment:
@RishabhJain2018 I think you can easily handle this at client side. Just check for null URL then use ng-if to hide those links. @deshraj  I think this would be a better solution.